oak-search/src/test/java/org/apache/jackrabbit/oak/plugins/index/IndexQueryCommonTest.java

```diff
@@ -142,6 +143,24 @@ public void descendantTest() throws Exception {
         });
     }
 
+    @Test
+    public void descendantTestWIthIndexTag() throws Exception {
+        Tree test = root.getTree("/").addChild("test");
+        test.addChild("a");
+        test.addChild("b");
+        root.commit();
+
+        assertEventually(() -> {
+            Iterator<String> result = executeQuery(
+                    "select [jcr:path] from [nt:base] where isdescendantnode('/test') option (index tag x)",
+                    "JCR-SQL2").iterator();
+            assertTrue(result.hasNext());
+            assertEquals("/test/a", result.next());
```

Comment:
I believe the test would pass even if no index is used, or the wrong index. I believe it would be better if the test verifies the right index is used, by executing "explain select" and then checking the plan.

I wonder which Jira issue this test refers to?